CodeExamples.md

```diff
@@ -578,10 +597,12 @@ Lets check that the download URL was successfully received
       console.log(fileData)
     } catch (e) {
       LoggerInstance.error('Download failed', e)
-      assert.fail('Download failed')
+      assert.fail('Download failed') ///
```

Comment:
Is this required as a marker for code examples?

Reply:
no i've tried to remove that from the md file, but we need a different approach, will update it